XenModel/Actions/MultipleAction.cs

```diff
@@ -127,15 +143,26 @@ private void SubActionChanged(ActionBase sender)
             {
                 SubActionTitle = subAction.Title;
                 SubActionDescription = subAction.Description;
+                RecalculatePercentComplete();
                 OnChanged();
             }
         }
 
+        protected virtual void RecalculatePercentComplete()

```

Comment:
Yes, indeed, that would work only if we had the percentage delta.
In any case, the PR looks ok now, I'm merging it.